src/mozillavpn.cpp

```diff
@@ -1381,6 +1381,11 @@ void MozillaVPN::registerUrlOpenerLabels() {
   uo->registerUrlLabel("upgradeToAnnualUrl", []() -> QString {
     return Constants::upgradeToAnnualUrl();
   });
+
+  uo->registerUrlLabel("sumoDevices", []() -> QString {
+    return "https://support.mozilla.org/en-US/kb/"
```

Comment:
return "https://support.mozilla.org/kb/" 

Moz.org will route the user to the correct locale based on their browser language settings